src/main/java/jenkins/plugins/publish_over_ssh/BapSshClient.java

```diff
@@ -41,11 +37,9 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
It's strange to import all by * and later male a particular import. In general it's netter to make particular imports